lizard_auth_server/testsettings.py

```diff
@@ -178,3 +178,23 @@
 DEFAULT_FROM_EMAIL = "noreply@nelen-schuurmans.nl"
 EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
 OIDC_USERINFO = "lizard_auth_server.oidc.userinfo"
+
+# # These settings are needed for the cognito backend, but enabling them here
+# # wreaks everything as the IDs aren't real, of course. So.... we'll do it in
+# # the tests.
+# AUTHENTICATION_BACKENDS = [
+#     "lizard_auth_server.backends.CognitoBackend",
```

Comment:
So the SSO will do:

AD Backend, Cognito Backend, ModelBackend ?

Reply:
Yes. Though the modelbackend might not do much in the end. I've included it here to be able to log in as a local admin on my localhost as cognito uses the staging db :-)